dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/context/annotation/DubboConfigBindingRegistrar.java

```diff
@@ -85,7 +86,7 @@ private void registerDubboConfigBeans(String prefix,
                                           boolean multiple,
                                           BeanDefinitionRegistry registry) {
 
-        Map<String, String> properties = getSubProperties(environment.getPropertySources(), prefix);
+        Map<String, Object> properties = getSubProperties(environment.getPropertySources(), prefix);
```

Comment:
why modify type?

Reply:
It's better for binding, `PropertySource` and `PropertyValue` both depend on `Map<String,Object>`